apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -25,6 +26,10 @@ export function LoginForm({}: Props) {
     }
   >((data) => api.post(`/v1/auth/login`, data));
 
+  const { isFromVercel, code, next } = useVercelIntegration();
```

Comment:
maybe this flag and params should be moved to a separate hook? as we don't use the other logic here (like query etc)

Reply:
I personally struggle to understand what you are proposing as this params are needed for the signup and github links that are used in the component rendering of the LoginForm. Would you mind to clarify a bit what you are proposing? 🤔

Reply:
I'm not sure about splitting it into two different hooks, as it will be used in only a couple of places and won't be used in any other functionalities. Having it in a single hook isolates it to that particular use case. 🤔 
Wdyt? will it be still better to split them?

Reply:
That is not required, however like I said nothing is for free, so React will need to call some hooks, state, effects that are inside the `useVercelIntegration` hook. 
I've also called it a tiny suggestion because it won't impact the rendering phase in any visible manner.